glusterd2/commands/volumes/volume-smartvol-txn.go

```diff
@@ -60,7 +60,13 @@ func txnPrepareBricks(c transaction.TxnCtx) error {
 			}
 
 			// Make Filesystem
-			err = deviceutils.MakeXfs(b.DevicePath)
+			mkfsOpts := []string{}
+			if b.Type == "arbiter" {
+				mkfsOpts = []string{"-i", "size=512", "-n", "size=8192", "-i", "maxpct=0", b.DevicePath}
+			} else {
+				mkfsOpts = []string{"-i", "size=512", "-n", "size=8192", b.DevicePath}
+			}
+			err = deviceutils.MakeXfs(mkfsOpts...)
```

Comment:
Mixing devicePath with options is not looking good, retain first argument as devicePath  `deviceutils.MakeXfs(b.DevicePath, mkfsOpts...)`

Reply:
done